content/talks/css-rad.md

```diff
@@ -8,7 +8,7 @@ image:
   alt: Miriam Suzanne speaking at a conference
 slides: https://sliiides.netlify.com/css-is-rad/
 tags:
-  - Code
+  - Design
```

Comment:
@mirisuzanne Are the "code video" tags below in the events list being used for anything?

Reply:
I don't think so. Probably a copy-paste hold-over…